middleware/router.go

```diff
@@ -56,11 +61,6 @@ func NewRouter(router *gin.Engine) *gin.Engine {
 		apiGroup.GET("/users", api.GetUsers)
 	}
 
-	// HTML Templates (e.g Status page)
-	router.LoadHTMLGlob("./views/**/*")
-	router.GET("/status", httpTemplates.StatusPageResponse(), StatusPageMiddleware())
-	router.GET("/docs", DocumentationPageMiddleware())
-
```

Comment:
Don't you need this page anymore?

Reply:
Ah, I think I can see, you deleted it and did on some domain

Reply:
They are moved upper, the docs is also changed. Header component in views:
```
                        <li><a href="https://docs.goblitz.org/" class="text-white ml-8">Documentation</a></li>
```
And the middleware/router.go. You should see in branch that:
```
	// Frontend & Static Files Handling
	router.LoadHTMLGlob("./views/**/*.html")

	router.Use(static.Serve("/assets", static.LocalFile("./public/assets", true)))
	if helper.CheckFileNotExists("./public/index.html") {
		router.GET("/", WelcomePageMiddleware())
		router.GET("/favicon.ico", func(c *gin.Context) {
			c.String(http.StatusNoContent, "")
		})
	} else {
		router.Use(static.Serve("/", static.LocalFile("./public/", true)))
	}

	helper.ServePageAssets(router)

	// HTML Template generated pages
	router.GET("/status", httpTemplates.StatusPageResponse(), StatusPageMiddleware())
```
The router.LoadHTMLGlob(".views/**/*") is a bug, which makes trouble